pandasai/prompts/explain_prompt.py

```diff
@@ -0,0 +1,30 @@
+""" Prompt to explain solution generated
+Based on the last conversation you generated the code. 
+Can you explain briefly for non technical person on how you came up with code 
+without explaining pandas library?
+"""
+
+
+from .base import Prompt
+
+
+class ExplainPrompt(Prompt):
+    """Prompt to get clarification questions"""
+
+    text: str = """
+The previous conversation we had
+
+<Conversation>
+{conversation}
+</Conversation>
+
+Based on the last conversation you generated the code. 
+
+<Code>
+{code}
+</Code
+
+Can you explain briefly for non technical person on how you came up with code 
+without explaining pandas library?
```

Comment:
Let's try with "Explain how you came up with code for non-technical people without mentioning technical details or mentioning the libraries used".